crates/fluvio-spu/src/services/public/tests/mod.rs

```diff
@@ -10,19 +10,23 @@ use fluvio_controlplane::spu_api::update_smartmodule::SmartModule;
 use fluvio_controlplane_metadata::smartmodule::{
     SmartModuleSpec, SmartModuleWasm, SmartModuleWasmFormat,
 };
+use fluvio_socket::AsyncResponse;
+use fluvio_spu_schema::server::stream_fetch::StreamFetchRequest;
 use fluvio_protocol::{
     fixture::BatchProducer,
     record::{RecordData, Record, RecordSet, Batch, RawRecords},
     ByteBuf,
 };
 use fluvio_storage::{FileReplica, ReplicaStorage};
+use futures_util::StreamExt;
```

Comment:
nit.  move this in third party libraries

Reply:
the order? fixed!